net-perf/src/components/loadFile/LoadFile.js

```diff
@@ -1,69 +1,43 @@
 import React, { useState, useEffect, useRef } from 'react';
+import Uploadbtn from './Uploadbtn';
 
 const LoadFile = props => {
-	const inputRef = useRef(null);
+	const [filenames, setFilenames] = useState({ 0: '', 1: '', 2: '' });
 
 	const onExecuteClick = e => {
 		props.setOnRunning(true);
 	};
 
-	const [filenames, setFilenames] = useState(['', '', '']);
-
 	const chooseFile = e => {
-		console.log(
-			'hhhh제발 좀 되라!!제발 좀 되라!!제발 좀 되라!!제발 좀 되라!!제발 좀 되라!!h1111111',
-		);
-		let file = e.target.files[0];
-		// var file = inputRef.current.file[0];
-		setFilenames([file.name, '', '']);
-		console.log(
-			'hhhh제발 좀 되라!!제발 좀 되라!!제발 좀 되라!!제발 좀 되라!!제발 좀 되라!!h222222',
-		);
+		switch (e.target.id) {
+			case '0':
+				setFilenames(prevState => ({
+					...prevState,
+					0: e.target.files[0].name,
+				}));
+				break;
+			case '1':
+				setFilenames(prevState => ({
+					...prevState,
+					1: e.target.files[0].name,
+				}));
+				break;
+			case '2':
+				setFilenames(prevState => ({
+					...prevState,
+					2: e.target.files[0].name,
+				}));
+				break;
+		}
```

Comment:
```suggestion
               let index = parseInt(e.target.id);
               setFilenames(prevState => ({
					...prevState,
					index : e.target.files[index].name,
				}));
```

`e.target.id`을 index 변수로 선언하게 되면 switch 문 없이 이렇게 줄일 수 있을 것 같습니다.